src/test/scala/org/kidsfirstdrc/dwh/testutils/ClassGeneratorMain.scala

```diff
@@ -15,13 +17,28 @@ object ClassGeneratorMain extends App with WithSparkSession {
     .withColumn("sampleId", lit("id"))
     .withColumn("genotypes", array(struct(col("sampleId") as "sampleId")))
     .drop("sampleId")
-  val clinvarOutput = ImportClinVar.transform(clinvarInput)
 
+  clinvarInput
+    //.writeCLassFile("org.kidsfirstdrc.dwh.testutils.external","ClinvarInput", root)
 
-  ClassGenerator.writeCLassFile("org.kidsfirstdrc.dwh.testutils.external","ClinvarInput", clinvarInput, root)
-  ClassGenerator.writeCLassFile("org.kidsfirstdrc.dwh.testutils.external","ClinvarOutput", clinvarOutput, root)
+  ImportClinVar
+    .transform(clinvarInput)
+    //.writeCLassFile("org.kidsfirstdrc.dwh.testutils.external","ClinvarOutput", root)
 
-  val variants = spark.read.format("parquet").load("src/test/resources/variants/variants.parquet")
 
-  ClassGenerator.writeCLassFile("org.kidsfirstdrc.dwh.testutils.variant","Variant", variants, root)
+  spark.read.format("parquet").load("src/test/resources/variants/variants.parquet")
+    //.writeCLassFile("org.kidsfirstdrc.dwh.testutils.variant","Variant", root)
+
+
+  val orphanetPath = getClass.getResource("/raw/orphanet").getFile
+  val orphanetData = new ImportOrphanetJob(Environment.DEV).extract(orphanetPath)(spark)
+  orphanetData("product6").where(col("orpha_code") === 447)
```

Comment:
magic numbers

Reply:
I want to find exactly this id in my test data or the test should fail